tests/inherit_postcondition/module_src/mod.fz

```diff
@@ -31,7 +31,7 @@ public mod is
   ok(s String) => say s; true
 
   public post_accessing_result is
-    public n := 333
+    public n => 333
```

Comment:
This test checks that the post condition can read a field. Why does this need to be changed?

Reply:
I did not see this. I think best would be to have both: The original `n` and a new function `m` that is redefined as a field in `inh_module2`.